cassdegrees/templates/widgets/staff/rulescripts.html

```diff
@@ -334,8 +344,8 @@
         <div class="msg-warn" v-if="is_list_search">Searching in list mode,
             <a href="javascript:void(0);" @click="toggleListMode()">return to course mode</a></div>
 
-        <ul class="resources-list">
-            <template v-for="(resource, index) in sortedSelectedList">
+        <ul class="resources-list" v-if="!is_hidden_courses">
```

Comment:
`is_hidden_courses` doesn't seem like the best naming scheme here - `is_courses_view_hidden` maybe?